src/main/java/com/todoist/pojo/Utils.java

```diff
@@ -8,4 +13,26 @@ static boolean equals(Object a, Object b) {
     static int clamp(int val, int min, int max) {
         return val < min ? min : (val > max ? max : val);
     }
+
+    public static <T> Set<T> unmodifiableSet(Collection<? extends T> s) {
+        if (s != null) {
+            return Collections.unmodifiableSet(new HashSet<T>(s));
+        } else {
+            return Collections.emptySet();
```

Comment:
Makes sense! 👍 Fixed in 69bcb7801f4633880293f731a8f3feaa3c185f3e and also made the method package-private like the rest of the class.